sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md

```diff
@@ -23,8 +23,6 @@ in this folder, with the following tags:
 
 `--use=<path to autorest clone>`
```

Comment:
Remove this as well. The `yaml` section has the autorest version defined, so cloning the autorest repo is not required

Reply:
Simplified to "run `autorest readme.md`"